google-cloud-storage/src/main/java/com/google/cloud/storage/BlobInfo.java

```diff
@@ -795,17 +795,18 @@ public Long getMetageneration() {
     return metageneration;
   }
 
-  /** Returns the deletion time of the blob. */
+  /** Returns the deletion time of the blob expressed as the number of ms since the Unix epoch. */
```

Comment:
ms --> milliseconds
"Returns the deletion time of the blob" --> "Returns the time the blob was deleted"